server/src/main/java/com/objectcomputing/checkins/services/frozen_template_questions/FrozenTemplateQuestion.java

```diff
@@ -16,45 +16,46 @@
 import java.util.UUID;
 
 @Entity
-@Table(name = "feedback_request_questions")
-public class FeedbackRequestQuestion {
+@Table(name = "frozen_template_questions")
+public class FrozenTemplateQuestion {
     @Id
     @Column(name = "id")
     @AutoPopulated
     @TypeDef(type = DataType.STRING)
     @Schema(description = "unique id of the request question answer entry", required = true)
     private UUID id;
 
-    @Column(name = "question")
+    @Column(name = "frozen_template_id")
+    @NotNull
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of the versioned template (and by extension, feedback request) that question is attached to ", required = true)
+    private UUID frozenTemplateId;
+
+    @Column(name = "question_content")
     @NotBlank
     @TypeDef(type = DataType.STRING)
     @ColumnTransformer(
-            read =  "pgp_sym_decrypt(question::bytea,'${aes.key}')",
+            read =  "pgp_sym_decrypt(question_content::bytea,'${aes.key}')",
             write = "pgp_sym_encrypt(?,'${aes.key}')"
     )
     @Schema(description = "The question asked to the recipient", required = true)
-    private String question;
-
-    @Column(name = "request_id")
-    @NotNull
-    @TypeDef(type = DataType.STRING)
-    @Schema(description = "id of the feedback request the question answer pair is attached to", required = true)
-    private UUID requestId;
+    private String questionContent;
```

Comment:
`question_content` has been renamed to `question` in all the DB migrations for `TemplateQuestion`

We can use either `question_content` or `question`, but we should keep them the same, whichever we choose

Reply:
Frozen question field changed to 'question' for consistency